src/utils.ts

```diff
@@ -98,3 +99,23 @@ export function filterKeyByBase(
 
   return key[key.length - 1] !== "$";
 }
+
+export function stringify(value: any): string {
+  if (typeof value === "string") {
+    return JSON.stringify(value);
```

Comment:
this would be a breaking change i think

since it means people who were previously storing strings as-is, will now store them as their JSON equivalents (`"foo"` instead of `foo`)